DragonKitchen/DragonKitchen/Assets.xcassets/Image/Title.imageset/Contents.json

```
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "타이틀.png",
```

이미지명이 한글이면 clone이나 pull했을 때 제대로 출력되지 않는 이슈가 있어요!
아래처럼 영어 파일명으로 수정이 필요해보입니다

```suggestion
      "filename" : "title.png",
```